libethereum/LogFilter.h

```diff
@@ -26,6 +26,10 @@
 #include <libethcore/Common.h>
 #include "TransactionReceipt.h"
 
+#ifdef __INTEL_COMPILER
+#pragma warning(disable:1098)
```

Comment:
Please can you add a comment here on the RHS of the #pragma line, with the warning message being suppressed?  Thanks!